.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -214,10 +214,19 @@ data class Stage(val stageName: StageName, val specificBuilds: List<SpecificBuil
     val id = stageName.id
 }
 
-data class TestCoverage(val uuid: Int, val testType: TestType, val os: Os, val testJvmVersion: JvmVersion, val vendor: JvmVendor = JvmVendor.oracle, val buildJvmVersion: JvmVersion = JvmVersion.java11, val expectedBucketNumber: Int = 50, val withoutDependencies: Boolean = false) {
-
-    constructor(uuid: Int, testType: TestType, os: Os, testJvm: JvmCategory, buildJvmVersion: JvmVersion = JvmVersion.java11, expectedBucketNumber: Int = 50, withoutDependencies: Boolean = false) :
-        this(uuid, testType, os, testJvm.version, testJvm.vendor, buildJvmVersion, expectedBucketNumber, withoutDependencies)
+data class TestCoverage(val uuid: Int,
+                        val testType: TestType,
+                        val os: Os,
+                        val testJvmVersion: JvmVersion,
+                        val vendor: JvmVendor = JvmVendor.oracle,
+                        val buildJvmVersion: JvmVersion = JvmVersion.java11,
+                        val expectedBucketNumber: Int = 50,
+                        val withoutDependencies: Boolean = false,
+                        val testDistribution: Boolean = false
+) {
+
+    constructor(uuid: Int, testType: TestType, os: Os, testJvm: JvmCategory, buildJvmVersion: JvmVersion = JvmVersion.java11, expectedBucketNumber: Int = 50, withoutDependencies: Boolean = false, testDistribution: Boolean = false) :
+        this(uuid, testType, os, testJvm.version, testJvm.vendor, buildJvmVersion, expectedBucketNumber, withoutDependencies, testDistribution)
```

Comment:
Question: I know this was not added in this PR, but do we really need this additional constructor? There are defaults for many things already and we can use named arguments in the calls.

Reply:
Note that `JvmCategory` is a combination of `JvmVersion` and `JvmVendor`. `JvmCategory` is not `TestCoverage`'s property.

Reply:
I didn't notice. Thanks for explaining.